supervisor/store/repository.py

```diff
@@ -138,6 +178,25 @@ async def update(self) -> bool:
 
```

Comment:
```suggestion
        """Load addon repository."""
        await self._git.load()
    async def update(self) -> bool:
        """Update add-on repository.
        Returns True if the repository was updated.
        """
```
Sorry about the size of the suggestion, it won't let me comment direclty on code it has collapsed I guess. Anyway there's no point in running validate first in this class as defined, its always `True`